tests/test_types.py

```diff
@@ -1288,7 +1289,7 @@ def __bool__(self) -> bool:
         ('float_check', b't', ValidationError),
         ('uuid_check', 'ebcdab58-6eb8-46fb-a190-d07a33e9eac8', UUID('ebcdab58-6eb8-46fb-a190-d07a33e9eac8')),
         ('uuid_check', UUID('ebcdab58-6eb8-46fb-a190-d07a33e9eac8'), UUID('ebcdab58-6eb8-46fb-a190-d07a33e9eac8')),
-        ('uuid_check', b'ebcdab58-6eb8-46fb-a190-d07a33e9eac8', UUID('ebcdab58-6eb8-46fb-a190-d07a33e9eac8')),
+        ('uuid_check', b'ebcdab58-6eb8-46fb-a190-d07a33e9eac8', ValidationError),
```

Comment:
If I run a python script to test, I get an error. So I think it's normal to have a `ValidatorError`

```print(str(UUID(b'a6cc5730-2261-11ee-9c43-2eb5a363657c')))``` -> `TypeError: a bytes-like object is required, not 'str'`